src/test/java/competition/subsystems/shooterwheel/ShooterWheelCommandTest.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.shooterwheel;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.shooterwheel.commands.SpinningShooterWheelCommand;
+
+public class ShooterWheelCommandTest extends BaseCompetitionTest {
+    
+    @Test
+    public void testSpin() {
+        ShooterWheelSubsystem shooterWheel = this.injector.getInstance(ShooterWheelSubsystem.class);
+        SpinningShooterWheelCommand command = this.injector.getInstance(SpinningShooterWheelCommand.class);
+        command.initialize();
+        command.execute();
+
+        shooterWheel.changeTargetSpeed(100);
+    
+        assertEquals(shooterWheel.getTargetSpeed(), shooterWheel.neoMasterMotor.getVelocity(), 150);
```

Comment:
I don't think this is doing what you think it's doing, the `150` here is the margin for how close the numbers have to be to be considered equal. It's usually used for floating point numbers to provide a very small delta (see `testStop` below). 

The goal should be that the shooterWheel.neoMasterMotor pidGoal is now 100 right?